src/FormControl.js

```diff
@@ -32,7 +32,10 @@ const propTypes = {
    * <FormControl inputRef={ref => { this.input = ref; }} />
    * ```
    */
-  inputRef: PropTypes.func
+  inputRef: PropTypes.oneOfType([
```

Comment:
let's just go with `any` maybe

Reply:
no exposed type for those ref things, eh?